src/components/atoms/UserStatusDropdown/UserStatusDropdown.tsx

```diff
@@ -1,46 +1,52 @@
-import React, { useMemo } from "react";
+import React, { useMemo, useEffect } from "react";
 import { Dropdown, DropdownButton } from "react-bootstrap";
 
-import { USER_STATUSES } from "settings";
-
-import { useProfileStatus } from "hooks/useProfileStatus";
-
 import { UserStatus } from "types/User";
 
+import { useVenueUserStatuses } from "hooks/useVenueUserStatuses";
+
 import "./UserStatusDropdown.scss";
 
 export interface UserStatusDropdownProps {
-  userStatuses?: UserStatus[];
+  userStatuses: UserStatus[];
 }
 
 export const UserStatusDropdown: React.FC<UserStatusDropdownProps> = ({
   userStatuses,
 }) => {
-  const { status, changeUserStatus } = useProfileStatus();
+  const { userStatus, changeUserStatus } = useVenueUserStatuses();
 
-  const allUserStatuses = useMemo(
-    () => (userStatuses ? [...USER_STATUSES, ...userStatuses] : USER_STATUSES),
-    [userStatuses]
-  );
+  // This will check if the user status from the database exists in the venue user statuses and if it doesn't, it will fallback to the first one from the list.
+  useEffect(() => {
+    if (!userStatus) return;
+
+    const statusTexts = userStatuses.map((userStatus) => userStatus.status);
+
+    const defaultUserStatus = userStatuses[0].status;
```

Comment:
I'm curious if `userStatuses[0].status` is a default status we should display... 
maybe we'll better use default status from `settings` ? because the 1st status in the `userStatuses` array is not necessarily what we need.
![image](https://user-images.githubusercontent.com/63313732/125101039-c5f10780-e0a7-11eb-8cf8-3141221c8bff.png)
am I a noob then?.. 😐😅 

I guess it'd be more correct to say that I'm just online (our default status).

If we agree on that, seems that we don't need `defaultUserStatus` const and can do 
```
       changeUserStatus(userStatus);
```

as the `useVenueUserStatuses` hook will do that for us.

Reply:
Great callout. I can change the order and make "here for it" the first option and default.  I think that status [0] should def be default rather than "online" being hard coded

Reply:
Tried to change the order (by adding new one, re-typing over 1, deleting first position) and getting this unhelpful error lol 

![image](https://user-images.githubusercontent.com/17037335/125128752-51bc6100-e0b3-11eb-91a2-6a572596395c.png)

Reply:
(this is the very bottom of the page, I cant scroll any more and there is nothing about the error displayed even in html)

Reply:
I went back in, changed last status to `here for it 2`, changed status 1 to 'here for it`, saved, it worked. Then went back in, changed last status to `a n00b!` and saved, and it worked. So I'm unblocked here

Reply:
@denisdimitrov and @sunny-viktoryia  That's not a problem you need to fix for us at the code level. If the first two statuses are not hard-coded, as an admin I can decide for myself what the first / default status should be. I'm going to call this resolved.

Reply:
(cc: @jonboutelle fyi)